Lesson5/auth2.py

```diff
@@ -0,0 +1,51 @@
+Users = {
+    'Feityanka': 'Clownnumber1',
+    'Yasos': 'Biba',
+    'Yazagit': 'Lera'
+}
+
+
+def get_name():
+    username = input('Enter your nickname for a registration.' "\n")
+    check_name(username)
+    return username
+
+
+def check_name(name):
+    if len(name) < 2:
+        print("Error. The name is too short. The name must contain more than 1 character.")
+        return None
```

Comment:
better to remove None, we can use just return. We return None when we dont specify anything